docs/learn/learn-extrinsics.md

```diff
@@ -48,6 +48,25 @@ Here are some key differences between the different types of extrinsics:
 - Inherents exist to address the need of adding some data to a block, whereas signed or unsigned
   transactions exist to potentially change the state of the blockchain.
 
+### Mortal and Immortal Extrinsics
+
+Extrinsics can be mortal (i.e. valid within a defined block interval) or immortal (i.e. always
+valid). By default and for security reasons, all extrinsics submitted using the
+[Polkadot-JS UI](../general/polkadotjs-ui.md) and
+[browser extension](../general/polkadotjs.md#polkadot-js-extension) will be mortal. This will avoid
```

Comment:
Does the extension actually matter here? It's just a signer, doesn't the Polkadot-JS App actually generate the extrinsic and the extension just signs it?

I just don't want users to think that anything signed with the browser extension is definitely mortal.

Reply:
Right, I will remove those tools and keep a general tone